core/sql/sqlcomp/PrivMgrPrivileges.cpp

```diff
@@ -5390,7 +5390,8 @@ PrivStatus ObjectPrivsMDTable::updateWhere(const std::string & setClause,
 //    <catalogName> "<schema_name>"."<object_name>", 
 //    object_type,
 //    object_owner, -- granteeID
-//    (select auth_db_name from AUTHS where auth_id = object_owner), --granteeName
+//    coalesce((select auth_db_name from AUTHS where auth_id = object_owner),
+//             'DB__ROOT') --granteeName
```

Comment:
I will investigate. If you prefer, I can use something different than DB__ROOT. I tried DB__ROOT as it seemed likely to be the most restrictive.